### ISSUE TITLE
RPC - transactions serialized with incorrect signature v value

### ISSUE THREAD

**Issue by kumavis** [ethcore/parity#4011]
### client:
```
Parity//v1.4.6-beta-5feccf1-20161205/x86_64-linux-gnu/rustc1.13.0
```
### problem:
RPC serialized transactions with incorrect signature v value

```
$ curl -H "Content-Type: application/json" -X POST --data '{"id":120,"jsonrpc":"2.0","method":"eth_getTransactionByHash","params":["0xe9e15dd4f1070ec30ca4bfbe70738e78b0bb7d126a512e7dc9b22df5b64af791"]}' https://mainnet.infura.io | jsonogram
├─ jsonrpc: 2.0
├─ result
│  ├─ blockHash: 0xc596cb892b649b4917da8c6b78611346d55daf7bcf4375da86a2d98810888e84
│  ├─ blockNumber: 0x2ca14a
│  ├─ creates
│  ├─ from: 0x56ce1399be2831f8a3f918a0408c05bbad658ef3
│  ├─ gas: 0x5208
│  ├─ gasPrice: 0x4e3b29200
│  ├─ hash: 0xe9e15dd4f1070ec30ca4bfbe70738e78b0bb7d126a512e7dc9b22df5b64af791
│  ├─ input: 0x
│  ├─ nonce: 0x9d
│  ├─ publicKey: 0x82a51957d31375120093359ab7beefe9d7d68579d6e40b99d85bff1283a687f8a88c0f3c0a1160f68df814a3bfe7f649a4c16f9b6dbdaf289e6dec93bf083287
│  ├─ r: 0x5d92c10b5789801d4ce0fc558eedc6e6cccbaf0105a7c1f909feabcedfe56cd9
│  ├─ raw: 0xf86d819d8504e3b2920082520894f4702bb51b8270729db362b0d4f82a56bdd66c6588120a871cc0020000801ba05d92c10b5789801d4ce0fc558eedc6e6cccbaf0105a7c1f909feabcedfe56cd9a072cc370fa5fd3b43c2ba4e9e70fea1b5e950b4261ab4274982d8ae15a3403a33
│  ├─ s: 0x72cc370fa5fd3b43c2ba4e9e70fea1b5e950b4261ab4274982d8ae15a3403a33
│  ├─ to: 0xf4702bb51b8270729db362b0d4f82a56bdd66c65
│  ├─ transactionIndex: 0x1
│  ├─ v: 0
│  └─ value: 0x120a871cc0020000
└─ id: 120
```
### expected behavior
v should be {27,28} or an [EIP155 value](https://github.com/ethereum/EIPs/issues/155)

**Comment by tomusdrw** [comment]
Thanks for excellent bug report:) The behaviour is already fixed on master, see previous issue: #3601 I will prepare a backports of those changes for `beta`

**Comment by kumavis** [comment]
thanks 👍
